nagasaki_2nd.html

```diff
@@ -0,0 +1,443 @@
+    <script type="text/javascript">
+      try {
+        Typekit.load();
+      } catch (e) {}
+    </script>
+
+    <meta property="og:url" content="https://railsgirls.com/nagasaki_2nd" />
+    <meta property="og:image" content="https://railsgirls.com/images/railsgirls-sq.png" />
+    <!--[if lt IE 9]>
+    <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <header>
+      <div class="container_12">
+        <a id="logo" href="/"><img src="images/railsgirls-logo.png" alt="Rails Girls" /></a>
+        <nav>
+          <a href="./events.html">Events</a>
+          <a href="./materials.html">Materials</a>
+          <a href="./press.html">Press</a>
+          <a href="https://railsgirls.tumblr.com/">Blog</a>
+          <a href="https://guides.railsgirls.com">Guides</a>
+        </nav>
+      </div>
+    </header>
+
+
+
+    <!--   How to change the picture & header for site
+  In the assets folder there is something called rg-header.psd. Be sure to download the fonts also. Name the file rg-yourcityname-header.png and your city picture rg-yourcityname.png. Add them to /images.
+
+  Steps to show on site
+  1. Go to style.css
+  2. Add two lines, always replacing 'yourcityname' with your actual city name.
+
+          .yourcityname#promo {
+            padding-left:340px;
+            background:url('../images/rg-yourcityname.png') no-repeat 0 10px,
+                          url('../images/separator.png') no-repeat center bottom;
+          }
+
+          .yourcityname#promo h1 {
+            background:url('../images/rg-yourcityname-header.png') no-repeat left top;
+          }
+
+          It will take a while before the changes show, so don't panic!
+
+  -->
+
+
+    <!--  Change the header text. You can include a local hello or something else that brings context. Remember to mention the workshop is free. Remember to change to wufoo form link and dates. -->
+
+    <div id="container" class="container_12 page clearfix">
+      <div class="grid_12 omega alpha">
+        <div id="promo" class="new nagasaki-2nd">
+          <h1>Rails Girls Nagasaki
+            <small>14th & 15th February 2025</small>
+          </h1>
+          <p>こんにちは世界！</p>
+          <p>長崎で 2 回目の Rails Girlsが開催されます! <br />
+            Ruby on Rails のすてきな世界を私達と一緒に体験しましょう！
+          </p>
+
+          <p><strong>Rails Girls Nagasaki 2nd の参加者を募集します。<br />
+              2日間のワークショップとなります。
+              無料のワークショップですので、お気軽にご参加ください。</strong></p>
+         <!-- <p><strong>2月下旬より参加者の募集を開始いたします。今しばらくお待ち下さい。</strong></p> -->
+
+          <p>
+            <strong>
+              応募を開始しました！！！<br/>
+              <a href="https://connpass.com/event/340051/" target="_blank">こちら</a>からご応募ください。<br/>
```

Comment:
```suggestion
              <a href="https://nagasakirb.connpass.com/event/340051/" target="_blank">こちら</a>からご応募ください。<br/>
```

connpassのURL、`nagasakirb` がサブドメインについていると思います。